### ISSUE TITLE
Feedback for /versioned_docs/version-0.1.0/advanced-topics/serialization/inetworkserializable.md

### ISSUE THREAD

**Issue by LukeStampfli** [Unity-Technologies/com.unity.multiplayer.docs#271]
This page does not mention that we do not support serialization of type hiearchies or typed serialization e.g.

```csharp
public class Tool : INetworkSerializable
{
//.....
}

public class BuildTool : Tool
{
//.....
}

```
Is something we do not support.

**Comment by Briancoughlin** [comment]
HAs been added to the backlog will be addressed in future update. Thanks for your contribution

**Comment by s-omeilia-unity** [comment]
Addressed with updates pushing to live with PR #676
